src/main/resources/liquibase/crop_changelog/17_4_0.xml

```diff
@@ -69,4 +70,36 @@
 			VALUES ('23', 'GERMPLASM_LABEL_PRINTING_PRESET', 'Excel template default', '{\"type\":\"LabelPrintingPreset\",\"selectedFields\":[[8240,41,8189,8256,1712,48,1709]],\"barcodeSetting\":{\"barcodeNeeded\":false,\"automaticBarcode\":false,\"barcodeFields\":[]},\"includeHeadings\":true,\"fileConfiguration\":{\"outputType\":\"xls\"}}');
        </sql>
     </changeSet>
+	
+	<changeSet author="corina" id="v17.4.0-4">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+					SELECT COUNT(*) > 0
+					FROM methods
+					WHERE `mid` IN (202,502) and mname LIKE 'Double  haploid%';
+			</sqlCheck>
+		</preConditions>
+		<comment>
+            Remove excess space between "Double" and "haploid" in method names. IBP-4238
+        </comment>
+		<sql dbms="mysql" splitStatements="false">
+			UPDATE methods set mname=REPLACE(mname,'Double  haploid','Double haploid') WHERE `mid` IN (202,502);
+		</sql>
+	</changeSet>
+	
+	<changeSet author="corina" id="v17.4.0-5">
```

Comment:
@mchabito The changesets looks good to me. I wonder if there are GEN methods with wrong mprgn? and if they are, will they be fixed and how?